migrations/3_monetary_policy.js

```diff
@@ -86,7 +94,9 @@ async function migration(deployer, network, accounts) {
     Bond.address,
     Share.address,
     Oracle.address,
+    Oracle.address,
```

Comment:
Please deploy two oracle with `1 day`, `1 hour` period each. 
And use these contract as `seigniorageOracle`, `bondOracle`.

Reply:
@fr-jerry-smith Thanks for the feedback. I've made another commit. Please check if it looks good